### PR TITLE
Pv2 1072 create a web endpoint to trigger month end events for manual testing

### PR DESCRIPTION
**Pull request by samuel-dev-dba**

### CODE REVIEW

File: src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Infrastructure/EndpointNames.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Payments.PeriodEnd.TestEndpoint.Infrastructure
+{
+    public class EndpointNames
+    {
+        public const string ProviderPayments = "sfa-das-payments-providerpayments";
```
Comment (carlbm) [inline src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Infrastructure/EndpointNames.cs]: Shouldn't this be a config setting?

File: src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false, 
+    "anonymousAuthentication": true, 
+    "iisExpress": {
+      "applicationUrl": "http://localhost:6738",
```
Comment (carlbm) [inline src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Properties/launchSettings.json]: Shouldn't the port be a config setting?
Also, not sure that localhost will work? Maybe 0.0.0.0 if it doesn't

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Properties/launchSettings.json]: This only applies to the dev environment

File: src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Infrastructure/EndpointNames.cs (hunk 1 above)
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.PeriodEnd.TestEndpoint/Infrastructure/EndpointNames.cs]: done

**Comment by samuel-dev-dba** [comment]
No Need to merge to master will be manually deployed when required.

**Pull request closed without merging**